app/views/newspaper_works/base/_attribute_rows.html.erb

```diff
@@ -0,0 +1,36 @@
+<%# newspaper_core_metadata attributes %>
+<%= presenter.attribute_to_html(:date_modified, label: t('hyrax.base.show.last_modified'), html_dl: true) %>
+<%= presenter.attribute_to_html(:alternative_title, render_as: :faceted, html_dl: true) %>
+<%= presenter.attribute_to_html(:genre, render_as: :faceted, html_dl: true) %>
+<%= presenter.attribute_to_html(:place_of_publication_label, render_as: :faceted, html_dl: true) %>
+<%= presenter.attribute_to_html(:issn, render_as: :faceted, html_dl: true) %>
+<%= presenter.attribute_to_html(:lccn, render_as: :faceted, html_dl: true) %>
+<%= presenter.attribute_to_html(:oclcnum, render_as: :faceted, html_dl: true) %>
+<%= presenter.attribute_to_html(:held_by, render_as: :faceted, html_dl: true) %>
+<%# newspaper_article attributes %>
+<%= presenter.attribute_to_html(:author, render_as: :faceted, html_dl: true) %>
+<%= presenter.attribute_to_html(:photographer, render_as: :faceted, html_dl: true) %>
+<%= presenter.attribute_to_html(:volume, render_as: :faceted, html_dl: true) %>
+<%= presenter.attribute_to_html(:issue_number, render_as: :faceted, html_dl: true) %>
+<%= presenter.attribute_to_html(:geographic_coverage, render_as: :faceted, html_dl: true) %>
+<%= presenter.attribute_to_html(:extent, render_as: :faceted, html_dl: true) %>
+<%= presenter.attribute_to_html(:publication_date, render_as: :faceted, html_dl: true) %>
+<%# newspaper_page attributes %>
+<%= presenter.attribute_to_html(:height, render_as: :faceted, html_dl: true) %>
+<%= presenter.attribute_to_html(:width, render_as: :faceted, html_dl: true) %>
+<%# newspaper_title attributes %>
+<%= presenter.attribute_to_html(:edition, render_as: :faceted, html_dl: true) %>
+<%= presenter.attribute_to_html(:frequency, render_as: :faceted, html_dl: true) %>
+<%= presenter.attribute_to_html(:preceded_by, render_as: :faceted, html_dl: true) %>
+<%= presenter.attribute_to_html(:succeeded_by, render_as: :faceted, html_dl: true) %>
+<%= presenter.attribute_to_html(:publication_date_start, html_dl: true) %>
```

Comment:
Can we try and render `publication_date_start` and `publication_date_end` as human-readable dates? These are currently displaying like `YYYY-MM-DDT00:00:00Z`

Reply:
This should be working in the final app. I'm not sure how you're configured, but the date format is defined in the hyrax initializer created by the Hyrax generator.

See:

https://github.com/samvera/hyrax/blob/master/app/renderers/hyrax/renderers/date_attribute_renderer.rb

and 

https://github.com/samvera/hyrax/blob/b5c6605917c01c51b4504f7e27999146fc584713/lib/generators/hyrax/templates/config/initializers/hyrax.rb#L271